hr_infraction/models/hr_employee.py

```diff
@@ -0,0 +1,13 @@
+from openerp import models, fields
+
+
+class HrEmployee(models.Model):
+
+    _name = 'hr.employee'
+    _inherit = 'hr.employee'
+    infraction_ids = fields.One2many(
+        comodel_name='hr.infraction',
```

Comment:
No need to state comodel_name and inverse_name, you can just use One2Many('hr.infraction', 'employee_id',...